demo/packaged/sql/03-feature-engineering.sql

```diff
@@ -0,0 +1,151 @@
+-- Use Case: Real-Time Feature Engineering (feature-engineering)
+--
+-- Compute complex ML features over an unbounded data stream using bounded memory.
+--
+-- We implement several typical ML features in SQL and evaluate them over a real-time data stream:
+-- * Data enrichment using ASOF JOIN
+-- * Rolling aggregate over a 30-day time window
+-- * Global aggregates over the entire event history
+--
+-- As you browse through the code, note how complex features are implemented by composing simple
+-- SQL views.
+--
+-- Try it!
+-- * ▶️ Start the demo to generate and process a stream of 1 billion random credit card transactions.
+-- * 📈 Check the Performance tab: as Feldera processes millions of incoming transactions,
```

Comment:
One more suggestion. Can we use multi-line comments (/* */)? It looks less distracting in our UI.